src/main/java/Joseph/Tasks/Task.java

```diff
@@ -0,0 +1,25 @@
+package Joseph.Tasks;
+
+public abstract class Task {
+    private final String desc;
+    private boolean isDone;
+    public Task(String desc) {
+        this.desc = desc;
+        this.isDone = false;
+    }
+    public String getDone() {
+        return isDone ? "X" : " ";
+    }
+    public String getDesc() {
+        return this.desc;
+    }
+    public String getDetails() {
+        return this.desc;
+    }
```

Comment:
It seems that these 2 methods do the same thing, perhaps you could remove the getDetails method?

Reply:
oh thanks i missed that

Reply:
I actually meant for getDesc to return just the description, while getDetails would return the description along with the due for Deadlines and the timings for Events (I override getDetails in the Deadline and Event classes), but I can see how it can be misleading. Thanks!